### PR TITLE
Update w11

### PR DESCRIPTION
**Pull request by LianeHughes**
Updates required for week 11 (to go live by end of next week)

### CODE REVIEW

File: data/events.json [hunk 1]
```diff
@@ -56,6 +80,31 @@
             "event_url": "https://www.scilifelab.se/event/embl-scilifelab-data-science-workshop/",
             "description": "We would like to cordially invite members of the SciLifeLab and EMBL Communities to this workshop! This event will bring together researchers, scientific and technical staff at the EMBL and SciLifeLab, to make new acquaintances and collaborations and to learn and be inspired by each other.<br><br>The workshop program starts at 09:00 on May 15 and ends after lunch on May 16.<br><br>The workshop will be a hybrid event enabling participation to the wider community at all EMBL and SciLifeLab sites.<br><br>Looking forward to meeting you in Uppsala in May!"
         },
+        {
+            "title": "RDM Training and Tools working group",
+            "type": "Webinar",
+            "date_start": "2023-04-27",
+            "time_start": "10:00",
+            "date_end": "2023-04-27",
```
Comment (senthil10) [inline data/events.json:88]:
```suggestion
```

File: data/events.json [hunk 2]
```diff
@@ -56,6 +80,31 @@
...
+            "event_url": "https://www.lyyti.fi/reg/ttwg8",
+            "registration_url": "https://www.lyyti.fi/reg/ttwg8",
+            "description": "This session we will be joined by Ann-Charlotte Sonnhamer from the Data Centre, who will talk about risk assessment procedures and what to take into consideration when assessing tools.<br><br>After Ann-Charlottes presentation we will have an open discussion on what RDM tools are being considered at the different sites, what the Data Centre can offer and how we can promote the use of tools by the Data Centre."
+        },
+        {
+            "title": "NBIS Weekly Bioinformatics Drop-in",
+            "type": "Support session",
+            "date_start": "2023-04-25",
+            "time_start": "14:00",
+            "date_end": "2023-04-25",
```
Comment (senthil10) [inline data/events.json:101]:
```suggestion
```

File: data/events.json [hunk 3]
```diff
@@ -117,6 +166,18 @@
             "registration_url": "https://docs.google.com/forms/d/e/1FAIpQLSezMsYRvJ0F-GKFKL1cPG6TjxsbtksutOjUTxgcyyL_WXhTXA/viewform",
             "description": "National course open for PhD students, postdocs, researchers and other employees within all Swedish universities. The course is geared towards life scientists wanting to take the first steps towards a more systematic and reproducible approach to analysing and managing research data. Due to limited space the course can accommodate a maximum of 25 participants. If we receive more applications, participants will be selected based on several criteria. Selection criteria include correct entry requirements, motivation to attend the course as well as gender and geographical balance.<br><br><b>Fee:</b> 1500 SEK paid by invoice to NBIS. This includes lunches, coffee and snacks. Please note that NBIS cannot invoice individuals."
         },
+        {
+            "title": "NBIS Weekly Bioinformatics Drop-in",
+            "type": "Support session",
+            "date_start": "2023-04-18",
+            "time_start": "14:00",
+            "date_end": "2023-04-18",
```
Comment (senthil10) [inline data/events.json:174]:
```suggestion
```

File: data/events.json [hunk 4]
```diff
@@ -141,6 +202,42 @@
             "registration_url": "https://forms.gle/D2GFZJUVBE5FswjD7",
             "description": "This course aims to help researchers to visualize their data in different ways using R. This course will teach how to produce publication grade figures using R. A part of this course is also about making interactive plots that the researchers can view and share in a web-server to make interactive visualizations of their data."
         },
+        {
+            "title": "NBIS Weekly Bioinformatics Drop-in",
+            "type": "Support session",
+            "date_start": "2023-04-11",
+            "time_start": "14:00",
+            "date_end": "2023-04-11",
```
Comment (senthil10) [inline data/events.json:210]:
```suggestion
```

File: data/events.json [hunk 5]
```diff
@@ -141,6 +202,42 @@
...
+            "organisers": "NBIS",
+            "event_url": "https://nbis.se/events/",
+            "description": "Bioinformatics Drop-in is available weekly to anyone in the Swedish research community wanting to discuss bioinformatics questions with experts from NBIS."
+        },
+        {
+            "title": "NBIS Weekly Bioinformatics Drop-in",
+            "type": "Support session",
+            "date_start": "2023-04-04",
+            "time_start": "14:00",
+            "date_end": "2023-04-04",
```
Comment (senthil10) [inline data/events.json:222]:
```suggestion
```

File: data/events.json [hunk 6]
```diff
@@ -141,6 +202,42 @@
...
+            "organisers": "NBIS",
+            "event_url": "https://nbis.se/events/",
+            "description": "Bioinformatics Drop-in is available weekly to anyone in the Swedish research community wanting to discuss bioinformatics questions with experts from NBIS."
+        },
+        {
+            "title": "NBIS Weekly Bioinformatics Drop-in",
+            "type": "Support session",
+            "date_start": "2023-03-28",
+            "time_start": "14:00",
+            "date_end": "2023-03-28",
```
Comment (senthil10) [inline data/events.json:234]:
```suggestion
```

**Review by senthil10 — COMMENTED** [review]
Looks better without date_end for single day events

**Comment by LianeHughes** [comment]
committed removal of end date for single day events

**Pull request merged**